app/models/header_lookup.rb

```diff
@@ -1,14 +1,14 @@
 class HeaderLookup
-  MEMOIZED_MESH_FILE = "memoized_mesh.txt"
-  MEMOIZED_LCSH_FILE = "memoized_lcsh.txt"
-  MEMOIZED_LCNAF_FILE = "memoized_lcnaf.txt"
+  MEMOIZED_MESH_FILE = "memoized_mesh.json"
```

Comment:
We can probably remove the memoized files that were text formatted as Ruby hashes, but I don't think it makes much of a difference one way or the other.

Reply:
Removing them is good and cleaner. They will still be in version control anyway.